core/node/main/config.hpp

```diff
@@ -47,6 +47,8 @@ namespace fc::node {
     /** Node default key path */
     boost::optional<std::string> wallet_default_key_path;
 
+    size_t mpool_bls_cache_size{1000};
```

Comment:
```suggestion
    constexpr size_t mpool_bls_cache_size{1000};
```

Reply:
no, that's mutable config field with default value

Reply:
Ok